src/components/EggheadPlayer/props.js

```diff
@@ -30,6 +32,7 @@ export const propTypes = {
   onEnded: func,
   onError: func,
   onDuration: func,
+  onMuted: func,
```

Comment:
added this before I realized you could just listen for events in the player config which is way easier imo